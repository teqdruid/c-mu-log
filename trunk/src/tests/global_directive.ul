//global directives
// no support is currently supplied for global directives.

@global_directive("global directive not supported");

main(){AND:
	@print("this is not a global directive and is supported");
      }		
