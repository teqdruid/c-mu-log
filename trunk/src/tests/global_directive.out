Fatal error: exception Failure("Unsupported global directive")
